internal/app/go-filecoin/plumbing/msg/previewer.go

```diff
@@ -43,32 +43,5 @@ func NewPreviewer(chainReader previewerChainReader, cst cbor.IpldStore, bs bstor
 
 // Preview sends a read-only message to an actor.
 func (p *Previewer) Preview(ctx context.Context, optFrom, to address.Address, method abi.MethodNum, params ...interface{}) (gas.Unit, error) {
```

Comment:
We should either delete this method altogether or panic. Probably no one is going to believe the zero, but it's better to be broken than wrong.